packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -14,17 +17,25 @@ export default class QuanticSearchBox extends LightningElement {
     suggestions: [],
     value: '',
   };
-
   /** @type {any} */
   get suggestions() {
-    return this.state.suggestions.map((s, index) => ({ key: index, value: s.rawValue }));
+    return this.state.suggestions.map((s, index) => ({
+      key: index,
+      value: s.highlightedValue,
+    }));
   }
-
   /** @type {string} */
   @api engineId;
   /** @type {string} */
-  @api placeholder = 'Search';
+  @api placeholder = 'Search...';
+   /** @type {boolean} */
+   withoutSubmitButton = false;
```

Comment:
The `@api` annotation is missing for `withoutSubmitButton` otherwise users won't be able to set the attribute.

Same thing for `numberOfSuggestions`. I guess it should be public as well.

Reply:
done 👍